src/ExtensionRoot.ts

```diff
@@ -83,10 +83,7 @@ export class ExtensionRoot {
         const workspacesCommonId = Guid.create();
 
         this._reporter = new TelemetryReporter(
-            packageJsonContent[`name`],
-            extensionVersion,
-            packageJsonContent[`aiKey`],
-            /*firstParty*/ true);
+            packageJsonContent[`aiKey`]);
```

Comment:
🙏💡 thank you for this, it will be worth to share internally the implementation result and changes constituting in this PR alters any behaviour? Thank you so much.

Reply:
Working on the doc, I will share it internally.